shared/app.js

```diff
@@ -73,7 +73,7 @@ module.exports = Backbone.Model.extend({
      * Initialize the `ClientRouter` on the client-side.
      */
     if (!isServer) {
-      var ClientRouter = this.options.ClientRouter || require(defaultRouterModule);
+      var ClientRouter = this.options.ClientRouter || this.modelUtils._requireAMD(defaultRouterModule);
```

Comment:
It worked because of this: https://github.com/rendrjs/rendr/pull/447/files#diff-a11874ca1cacb6a9ac55db82846df825L13